modules/peers.js

```diff
@@ -938,6 +937,26 @@ Peers.prototype.shared = {
 	getPeersCount() {
 		return library.logic.peers.list(true).length;
 	},
+
+	/**
+	 * Utility method to get peers count by filter.
+	 *
+	 * @param {Object} parameters - Object of all parameters
+	 * @param {string} parameters.ip - IP of the peer
+	 * @param {string} parameters.wsPort - WS Port of the peer
+	 * @param {string} parameters.httpPort - Web Socket Port of the peer
+	 * @param {string} parameters.os - OS of the peer
+	 * @param {string} parameters.version - Version the peer is running
+	 * @param {int} parameters.state - Peer State
+	 * @param {int} parameters.height - Current peer height
+	 * @param {string} parameters.broadhash - Peer broadhash
+	 * @returns {int} count
+	 * @todo Add description for the return value
+	 */
+	getPeersCountByFilter(parameters) {
+		parameters.normalized = true;
```

Comment:
@diego-G IMO we need to set the `normalized` flag to false to avoid unwanted peer properties in the peer list, as we are doing just the count of peers.